DataStructures/Dictionaries/CuckooHashTable.cs

```diff
@@ -114,7 +114,7 @@ private void _rehash()
         /// </summary>
         private void _rehash(int newCapacity)
         {
-            int primeCapacity = PRIMES.GetNextPrime(newCapacity);
+            _ = PRIMES.GetNextPrime(newCapacity);
```

Comment:
Maybe we could even remove that underline too

Reply:
Looking more closely at this, looks like this line and the PRIMES field can be removed, they are not used at all.